firefox-ios/Client/ContentBlocker/ContentBlocker.swift

```diff
@@ -212,19 +212,22 @@ extension ContentBlocker {
     }
 
     func removeAllRulesInStore(completion: @escaping () -> Void) {
+        let dispatchGroup = DispatchGroup()
+        
         ruleStore?.getAvailableContentRuleListIdentifiers { [weak self] available in
             guard let available = available else {
                 completion()
                 return
             }
-            let deferreds: [Deferred<Void>] = available.map { filename in
-                let result = Deferred<Void>()
+            
+            for filename in available {
+                dispatchGroup.enter()
                 self?.ruleStore?.removeContentRuleList(forIdentifier: filename) { _ in
-                    result.fill(())
+                    dispatchGroup.leave()
                 }
-                return result
             }
-            all(deferreds).uponQueue(.main) { _ in
+            
```

Comment:
@ddh4r4m Could you remove the trailing whitespaces on lines 216, 222, and 229 so that the linter tests pass?
https://github.com/mozilla-mobile/firefox-ios/pull/17915/checks?check_run_id=20110558114

It's been a couple weeks so once you do that and update the branch from main I can re-run the tests and then approve.

Reply:
Done